examples/active_support_helper.rb

```diff
@@ -0,0 +1,30 @@
+module ActiveRecord
+  class Base
+  end
+end
+
+module ActiveSupport
+  class TestCase < Minitest::Test
+    def setup(*args, &block)
+    end
+
+    # Remove describe method if present
+    class << self
```

Comment:
Upon further review I actually want to remove all this code.  In the end since I'm using :model I don't think any of it's needed.  In our test describe is not previously defined the way that mintiest-rails has to usually worry about it possibly being defined, so it's just noise.  In fact I don't even need to declare AR::Base either.  I think this make it much simpler perhaps to where it could just exist within the test, but I think I want to keep it separate.  I'll push and see what you think.